web/src/server/api/routers/scores.ts

```diff
@@ -177,8 +177,18 @@ export const scoresRouter = createTRPCRouter({
         },
       });
 
+      const tags = await ctx.prisma.$queryRaw<{ value: string }[]>`
```

Comment:
please parallelize with name query + add timestamp filter to imrpove performance. have a look at trace router for example